flight/Modules/Actuator/actuator.c

```diff
@@ -686,6 +692,11 @@ static bool set_channel(uint8_t mixer_channel, uint16_t value, const ActuatorSet
 		case ACTUATORSETTINGS_CHANNELTYPE_PWM:
 			PIOS_Servo_Set(actuatorSettings->ChannelAddr[mixer_channel], value);
 			return true;
+#if defined(PIOS_INCLUDE_ONESHOT)
```

Comment:
You mean, if PIOS_INCLUDE_ONESHOT isn't defined, the servo should be written to 0. There is a  fault handler at line 708. It generates SYSTEMALARMS_ALARM_ACTUATOR later.